README.md

```diff
@@ -36,6 +36,9 @@ npm install -g gltf-pipeline
 
 `gltf-pipeline -i model.glb -j`
 
+#### Converting a glTF to Draco glTF
+`gltf-pipeline.js -i model.gltf -d -s -o modelDraco.gltf`
```

Comment:
Ah never mind about this one.

Reply:
OK

Reply:
Will it stay required to run it with `-s`? I would like to embed the textures into the `.glb` file but it currently does not correctly do so when using it without `-s`.

Reply:
I was able to embed the textures correctly after using a `.gltf` instead of a `.glb` as input source. Using a `.glb` as input resulted in a `66kb` file instead of a `~3.3mb` file.

Reply:
Ok I see the problem now when using a glb input with `-d`. Perhaps a buffer is getting optimized away?

Good catch @TimvanScherpenzeel

Reply:
It appears that only the first buffer is being used, the other ones are dropped.

Broken:
<img width="317" alt="screen shot 2018-04-23 at 19 52 48" src="https://user-images.githubusercontent.com/11870513/39144181-5217ded8-4730-11e8-83d9-55de9faea87c.png">

Working:
<img width="334" alt="screen shot 2018-04-23 at 19 56 07" src="https://user-images.githubusercontent.com/11870513/39144220-6a80491a-4730-11e8-9cb9-c9bb30c22c6e.png">

Reply:
I'm having trouble reproducing the issue. With just a 2.0 branch I cannot open the DamagedHelmet.glb and output as a non-draco gltf file (or any file).

This is my command line:
node bin/gltf-pipeline.js -i ~/github/glTF-Sample-Models/2.0/DamagedHelmet/glTF-Binary/DamagedHelmet.glb 

TypeError: header.equals is not a function
    at getImageExtension (~/github/AnalyticalGraphicsInc/gltf-pipeline/lib/getImageExtension.js:18:16)
    at writeImage (~/github/AnalyticalGraphicsInc/gltf-pipeline/lib/writeResources.js:78:21)
    at ~/github/AnalyticalGraphicsInc/gltf-pipeline/lib/writeResources.js:48:23

Reply:
As an input example that reproduces the issue I've attached a .zip file containing the various outputs. The original `.gltf` input file (including seperate textures and `.bin`) can be found in `seperate-gltf`: [models.zip](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/files/1944328/models.zip)

`(embed-draco-glb)`
`node ./bin/gltf-pipeline.js -i /Users/TVS/Projects/vendor/gltf-pipeline/models/seperate-gltf/DamagedHelmet.gltf -d -b` results in a correctly outputted Draco `.glb` file (3.3 MB)

`(embed-glb)`
`node ./bin/gltf-pipeline.js -i /Users/TVS/Projects/vendor/gltf-pipeline/models/seperate-gltf/DamagedHelmet.gltf -b` results in a correctly outputted regular `.glb` file (3.8 MB)

`(embed-draco-broken-glb)`
`node ./bin/gltf-pipeline.js -i /Users/TVS/Projects/vendor/gltf-pipeline/models/embed-glb/DamagedHelmet.glb -d -b` results in a broken outputted `.glb` file (66 KB). Note that I'm using the `.glb` file as input.

Reply:
@TimvanScherpenzeel I fixed the issue. The bufferViews for inlined images were not being processed.

Reply:
Thanks! 👍